warehouse/query-core/src/main/java/datawave/mr/bulk/RfileResource.java

```diff
@@ -69,6 +70,9 @@ protected void init(final String tableName, final Set<Authorizations> auths, Col
         conf = new Configuration();
         
         AccumuloClient client = getClient();
+        if (client instanceof WrappedAccumuloClient) {
+            client = ((WrappedAccumuloClient) client).getReal();
+        }
         
         Properties clientProperties = client.properties();
         final String instanceName = clientProperties.getProperty(ClientProperty.INSTANCE_NAME.getKey());
```

Comment:
The issue here was that WrappedAccumuloClient's override for `properties()` is hardcoded to return null, so we threw NPE on line 78. A cleaner solution is probably to change WrappedAccumuloClient instead...i.e., in the [datawave-accumulo-utils](https://github.com/NationalSecurityAgency/datawave-accumulo-utils/blob/release/accumulo-2.0/src/main/java/datawave/webservice/common/connection/WrappedAccumuloClient.java#L222)